Hide back arrow in Firefox

brianpeiris opened the pull request:
Firefox Nightly doesn't render HTML content correctly at the moment. It shows the back arrow enlarged in the bottom right of the right eye in the DK2. This PR hides the arrow for desktop devices.

brianpeiris committed:
Hide back arrow on desktop devices.

borismus commented:
Thanks Brian, but a few things:
1. I'd rather address the actual problem. Why doesn't it render properly in Firefox Nightly?
2. If the rendering issue cannot be resolved now, please add a comment and indicate that it's a hack. Also, this should be FF-specific, not non-mobile specific.

brianpeiris committed:
Merge branch 'master' of https://github.com/borismus/webvr-boilerplate into bugfix/hide-arrow-on-desktop

brianpeiris committed:
Hide back arrow in firefox only

brianpeiris commented:
Just noticed this is broken in Firefox Nightly for Android as well. I'm not entirely sure why it's not rendering properly but I'm guessing it's just that they're still in the middle of fixing support for HTML content in VR.
I've updated the PR to make it Firefox specific and added a comment.
